client/dashboard/agency-overview/index.tsx

```diff
@@ -1,6 +1,6 @@
 import { Button } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
-import PageLayout from '../page-layout';
+import PageLayout from '../components/page-layout';
```

Comment:
Makes me wonder whether `client/dashboard/components/page-layout` is better for this kind of imports at some point (especially as we move things around)

Reply:
I think you meant `calypso/dashboard/components/...' but yeah I get your point. I think some devs will have the import auto-generated with relative paths by the IDEs, so we have to somehow enforce that (via eslint? not sure) if we want this. We can discuss it later :)